projects/metis/src/app/login/login.component.ts

```diff
@@ -89,7 +89,7 @@ export class LoginComponent extends DataPollingComponent implements OnInit {
           },
           (err: HttpErrorResponse) => {
             this.notification = errorNotification(
-              err.status === 406
+              [406, 401, undefined].includes(err.status)
```

Comment:
Again, don't we want to check for all 4xx statuses?

Reply:
The "not found" status is not the same as an unauthenticated status, so should be handled differently!